examples/random_pick/random_pick/launch/demo/gpd.launch

```diff
@@ -0,0 +1,77 @@
+<launch>
+
+  <arg name="device"                     default="0"
+        doc="target device to execute inference. [0:CPU]|1:GPU|2:VPU|3:FPGA" />
+  <arg name="plane_remove"               default="false"
+        doc="if remove plane from PCL inputs" />
+
+  <!-- Load hand geometry parameters -->  
+  <include file="$(find random_pick)/launch/demo/hand_geometry.launch">
+    <arg name="node" value="detect_grasps" />
+  </include>
+
+  <!-- Load classifier parameters -->
+  <include file="$(find gpd)/launch/caffe/classifier_15channels.launch">
+    <arg name="node" value="detect_grasps" />
+  </include>
+  
+	<node name="detect_grasps" pkg="gpd" type="detect_grasps" output="screen">
```

Comment:
Better to not mix tabs and spaces. Convert to spaces

Reply:
The tabs have been converted to spaces.